sterror/sterror.go

```diff
@@ -62,6 +59,7 @@ func (e Error) Error() string {
 		composedErrorString += colon + e.Op.String()
 	case e.Info != "":
 		composedErrorString += colon + e.Info.String()
+	default:
```

Comment:
The `default` could handle the situation where nothing is set (referring to Scope = "").